package.json

```diff
@@ -13,23 +17,27 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "babel-eslint": "^8.0.1",
+    "ensure-node-env": "^1.0.4",
     "eslint": "^4.10.0",
     "eslint-config-airbnb": "^16.1.0",
     "eslint-config-skyscanner": "^3.0.0",
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.4.0",
-    "jest": "^21.2.1",
-    "stylelint": "^8.2.0"
+    "jest": "^22.4.0",
+    "stylelint": "^9.1.1"
   },
   "peerDependencies": {
-    "stylelint": "^8.0.0"
+    "stylelint": "^9.1.1"
   },
   "dependencies": {
-    "stylelint-config-standard": "17.0.0",
+    "postcss-less": "^1.1.3",
```

Comment:
`postcss-less`?

Reply:
I was fixing these warnings:
```
npm WARN postcss-html@0.12.0 requires a peer of postcss-less@>=1.1.0 but none is installed. You must install peer dependencies yourself.
npm WARN postcss-html@0.12.0 requires a peer of postcss-scss@>=1.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN postcss-html@0.12.0 requires a peer of sugarss@>=1.0.0 but none is installed. You must install peer dependencies yourself.
```

Reply:
It looks like all of these should come with stylelint: https://github.com/stylelint/stylelint/blob/master/package.json#L63

Is that error caused by a conflict then?